templates/patterns/description_list/description_list.ui.patterns.yml

```diff
@@ -0,0 +1,44 @@
+description_list:
+  label: "Description list"
+  description: "Description list patter is used for the paragraph OpenEuropa Paragraphs Horizontal description list."
+  settings:
+    style:
```

Comment:
Missing description. I have style associated with background or general style: primary, secondary... Here is more an orientation, maybe?

Reply:
I'll rename it to Orientation. Layout seems to be a reversed word that breaks the pattern.